is/identity-server-deployment.yaml

```diff
@@ -49,6 +49,8 @@ spec:
           initialDelaySeconds: 250
           periodSeconds: 10
         imagePullPolicy: Always
+        securityContext:
+          runAsUser: 802
```

Comment:
@dmxunlimit not clear on this specific usage. Why uid `802`?

Reply:
We need to specify the runAsUser id as the id of wso2carbon which has the permission of rwx in the NFS mount, otherwise, there is permission denied error.
Is there any other better way to address this?